src/client/components/OrganizationPanel.js

```diff
@@ -1,24 +1,64 @@
-import React from 'react'
-import styled from 'styled-components'
-import { Text } from '@aragon/ui'
+import React, { useContext } from 'react'
 import CardWrapper from '../wrappers/styleWrappers/CardWrapper'
-
-const addMore = () => console.log('add more orgs')
+import { useProfile } from '../hooks'
+import { ModalContext } from '../wrappers/modal'
+import OrganizationTile from './OrganizationTile'
+import { open, removeItem } from '../stateManagers/modal'
+import { Text, theme } from '@aragon/ui'
+import styled from 'styled-components'
 
 const OrganizationPanel = () => {
+  const { organizations, viewMode } = useProfile()
+  const { dispatchModal } = useContext(ModalContext)
+
+  const organizationsNotEmpty = Object.keys(organizations).length > 0
+
+  const cardProps = {
+    title: 'Organizations',
+    addMore: organizationsNotEmpty
+      ? () => dispatchModal(open('organization'))
+      : null,
+    addSeparators: true,
+    viewMode,
+  }
+
   return (
-    <CardWrapper title="Organisations" addMore={addMore}>
-      <Center>
-        <Text color="grey">Coming Soon</Text>
-      </Center>
+    <CardWrapper {...cardProps}>
+      {organizationsNotEmpty ? (
+        Object.keys(organizations).map(id => (
+          <OrganizationTile
+            key={id}
+            organizationData={organizations[id]}
+            removeItem={() => dispatchModal(removeItem(id, 'organizations'))}
+          />
+        ))
+      ) : (
+        <Center>
+          <Text size="xlarge">You have no organizations</Text>
+          {!viewMode && (
+            <Text
+              css={`
+                cursor: pointer;
+              `}
+              size="small"
+              color={theme.accent}
+              onClick={() => dispatchModal(open('organization'))}
```

Comment:
Let's only add `onClick` handlers to `<a>` and `<button>`. I think aragonUI's `<Button>` would work nicely here. Then we can get rid of the `cursor: pointer`, too.

Reply:
Done.